content_delegate.js

```diff
@@ -227,11 +227,7 @@ ContentDelegate.prototype.handleSingleDocumentPageCheck = function() {
 ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
   // Save a copy of the page source, altered so that the "View Document"
   // button goes forward in the history instead of resubmitting the form.
-  let originalSubmit = document.forms[0].getAttribute('onsubmit');
```

Comment:
So the main difference here is that we will attach to the form of the event rather than just generically to the first form on the page. Is that what you're trying to do? Seems more precise, but definitely a change in behavior potentially.

Reply:
Yeah, they should be the same thing. I'm not entirely sure just by reading the code, but the functionality still seems to be working properly.

Reply:
I think the functionality that the `onsubmit` / `history.forward()` / `previousPageHtml` is dealing with is the ability to hit Back and have The Right Thing Happen. Is that what you tested, Mike? Or did you just test that downloads worked?

Reply:
I tested both, yeah. If you download a claim and then hit back, it works.

Reply:
Hrmmm....I wonder if (a) it is browser-specific (b) if this code is even necessary at all?